src/test/java/touro/chess/ClickEventHandlerTest.java

```diff
@@ -20,13 +21,20 @@ public static void beforeClass()
     public void onPieceClick()
     {
         //given
+        ClickEventHandler clickEventHandler = new ClickEventHandler();
+        Board board = new Board();
+        board.setUpBoard();
         Location location = new Location((int) 10.7, (int) 60.5);
-        Square square = new Square();
+        Square square = new Square(location);
+        KnightPiece knightPiece = new KnightPiece(location, PieceColor.Black);
+        square.setPiece(knightPiece);
+        mouseEvent.getSceneX() = mock(mouseEvent.getSceneX().class);
```

Comment:
this line is my attempt to fix the next line error but it is not working.

Reply:
You need to use a `doReturn()`

Reply:
You want to create a mock MouseEvent